### ISSUE TITLE
IPv4 has stopped working

### ISSUE THREAD

**Issue by thinkl33t** [tapchat/tapchat#30]
I've just updated tapchat, and now the process only attaches itself to an IPv6 port, not IPv4.

Runing lsof only gives me the following for tapchat:
node      26775  parag0n   11u  IPv6 25106647      0t0  TCP *:8067 (LISTEN)

Could be something to do with #14 being merged recently.

**Comment by codebutler** [comment]
I'm not having any problems with IPv4. What OS is this on?

**Comment by benley** [comment]
@thinkl33t, on every unix system I'm aware of, binding to a TCP6 socket without restricting it to a specific listen address will result in the socket being available as both ipv6 and ipv4. Are you certain that the issue is a lack of an open ipv4 socket and not some other sort of connectivity issue revealed by the new availability of a v6 socket?

I'm running tapchat on linux 3.11something, and my lsof output looks the same:

```
$ sudo lsof -i :8067
COMMAND  PID   USER   FD   TYPE     DEVICE SIZE/OFF NODE NAME
node    5805 benley   11u  IPv6 3905819976      0t0  TCP *:8067 (LISTEN)
```

... and I can definitely connect to tapchat from the internet via ipv4.

**Comment by mpontes** [comment]
@benley this is not true on FreeBSD, it ships with net.inet6.ip6.v6only = 1 by default.

**Comment by aquatix** [comment]
I have the same problem on Debian sid; it only listens on ipv6.
